telegram/_chat.py

```diff
@@ -2464,6 +2689,92 @@ async def forward_to(
             message_thread_id=message_thread_id,
         )
 
+    async def forwards_from(
+        self,
+        from_chat_id: Union[str, int],
+        message_ids: Sequence[int],
+        disable_notification: ODVInput[bool] = DEFAULT_NONE,
+        protect_content: ODVInput[bool] = DEFAULT_NONE,
+        message_thread_id: Optional[int] = None,
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
+        write_timeout: ODVInput[float] = DEFAULT_NONE,
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: Optional[JSONDict] = None,
+    ) -> Tuple["MessageId", ...]:
+        """Shortcut for::
+
+             await bot.forward_messages(chat_id=update.effective_chat.id, *args, **kwargs)
+
+        For the documentation of the arguments, please see :meth:`telegram.Bot.forward_messages`.
+
+        .. seealso:: :meth:`forward_to`, :meth:`forward_from`, :meth:`forwards_to`.
+
+        .. versionadded:: NEXT.VERSION
+
+        Returns:
+            Tuple[:class:`telegram.Message`]: On success, a tuple of ``MessageId`` of sent messages
```

Comment:
```suggestion
            Tuple[:class:`telegram.MessageID`]: On success, a tuple of :class:`~telegram.MessageId` of sent messages
```
Please also check in other places